about/our-web-team.php

```diff
@@ -63,18 +80,55 @@
+                                <h6 class="h6-with-small"><a href="#"> Nicholas Tindle</a><span
+                                            class="small text-silver-chalice">President | Team Member</span></h6>
+                                <p>Nicholas Tindle is a Computer Engineering student at UNT. He works in software
+                                    engineering and loves hackathons. You can generally find him wearing a hat and
+                                    probably a sweatshirt. He has a long history of collaboration with UNT Robotics as
+                                    the first president, a loyal advisor, and now Project Manager. He has also served as
+                                    an advisor to the Dean and is currently an officer of Engineering United. Nick has
+                                    helped host numerous events at the university over the years. In his professional
+                                    life, he works in data analysis, web development, and python scripting. </p>
                                 <ul class="list-inline-lg">
                                     <li>
-                                        <a href="https://www.linkedin.com/in/ntindle" class="icon icon-sm text-primary fa-linkedin"></a>
+                                        <a href="https://www.linkedin.com/in/ntindle"
+                                           class="icon icon-sm text-primary fa-linkedin"></a>
                                     </li>
                                     <li>
-                                        <a href="https://github.com/ntindle" class="icon icon-sm text-primary fa-github"></a>
+                                        <a href="https://github.com/ntindle"
+                                           class="icon icon-sm text-primary fa-github"></a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <!--Henry Legay-->
+                <div class="cell-lg-12 bio-area">
+                    <div class="range range-sm-middle">
+                        <div class="cell-md-3"><img src="/images/web-team-pics/henry-legay.jpg" alt="" width="360"
+                                                    height="404" class="img-responsive"/>
+                        </div>
+                        <div class="cell-md-6">
+                            <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
+                                <h6 class="h6-with-small"> Henry Legay<span
```

Comment:
Add the a tags back and have it link to itself
something like
<h6 id="your-name" class="h6-with-small"><a href="#your-name">Your Name</a>

Reply:
```<h6 id="your-name" class="h6-with-small"><a href="#your-name">Your Name</a>```